contents/blog/ab-testing-guide-for-engineers.md

```diff
@@ -0,0 +1,205 @@
+---
+date: 2023-07-04
+title: "A software engineer's guide to A/B testing"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ['lior-neu-ner']
+featuredImage: ../images/blog/experiment-hog.jpg
+featuredImageType: full
+category: Product growth
+tags: 
+  - Guides
+  - Product metrics
+---
+
+A/B testing enables you to be confident that your product changes are having their intended effects. Rather than making decisions based on gut feeling or intuition, A/B testing provides empirical data to help you remove bias from decision making.
+
+Companies like [Netflix](https://uxdesign.cc/how-netflix-does-a-b-testing-87df9f9bf57c), [Uber](https://www.uber.com/en-GB/blog/supercharging-a-b-testing-at-uber/) and [Booking.com](https://booking.ai/tagged/ab-testing) use A/B testing to ship hundreds of changes a day. In this post, we'll teach you basic concepts to get you started in A/B testing and supercharge your product growth!
```

Comment:
I think we can come back to this last, but this intro doesn't work for me. Can we lean into your credentials here? Afaik, you have a lot of experience doing A/B tests at Meta etc. (and on your own project), let's talk about that. Write it in the first person ("I will guide you through") rather than third person ("we'll teach you the basics").

Reply:
Maybe have a play around with this in an intro and throughout. Where possible, it's better to be opinionated. Drawing from your experience, and writing in first person, might lift the whole piece up a level.